packages/components/CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 ### Deprecations
 
 -   `Radio`: Deprecate 36px default size ([#66572](https://github.com/WordPress/gutenberg/pull/66572)).
+-   `UnitControl`: Deprecate 36px default size ([#66791](https://github.com/WordPress/gutenberg/pull/66791)).
```

Comment:
Let's merge `trunk` and move this back up to the Unreleased section.

Reply:
This is now resolved.